source/parameters/bend.md

```diff
@@ -0,0 +1,193 @@
+%---------------------------------------------------------------------------------------------------
+(s:bend.params)=
+## BendP: Bend Parameters
+
+The `BendParams` stores the parameters that characterize the shape of a [`Bend`](#s:bend) element. 
+The only relavent shape parameter that is not in the `BendParams` is the
+length `L` which is in the `LengthParams`.
+
+The parameters of this group are:
+```{code} yaml
+bend_type         - Type of bend. Default: SECTOR.
+angle             - Reference bend angle.
+g                 - Reference bend strength = 1/radius.
+bend_field_ref    - Reference bend field.
+L_chord           - Chord length.
+tilt_ref          - Reference tilt.
+e1                - Entrance end pole face rotation.
+e2                - Exit end pole face rotation.
+e1_rect           - Entrance end pole face rotation.
+e2_rect           - Exit end pole face rotation.
+edge_int1         - Entrance end fringe field integral.
+edge_int2         - Exit end fringe field integral
+```
+
+```{figure} figures/bend.svg
+:name: f:bend
+:width: 90%
+ 
+Bend geometry. Red dots are the entry and exit points that define the origin for the
+coordinate systems at the entry end {math}`(s_1, x_1)` and exit ends {math}`(s_2, x_2)` respectively.
+
+A) Bend geometry with positive bend angle. For the geometry shown,
+`g`, `angle`, `rho`, `e1`, `e2`, `e1_rect`, and `e2_rect` are all positive.
+
+B) Bend geometry with negative bend angle. For the geometry shown,
+`g`, `angle`, `rho`, `e1`, `e2`, `e1_rect`, and `e2_rect` are all negative.
+Note: The figures are drawn for zero `ref_tilt` where the rotation axis is parallel to the
+{math}`y`-axis.
+```
+
+In detail:
```

Comment:
It would be clearer if the detailed descriptions of these parameters appear in the same order as the list appearing in lines 11-22.

Reply:
The parameter list in lines 11-22 has been reordered.